### ISSUE TITLE
[Feature Request] Only logged in users can see the VODs

### ISSUE THREAD

**Issue by szotyo-sys** [Zibbp/ganymede#121]
**Idea**:  If the user is not logged in, they can't see nothing in the webpage except the login form.

**Problem**: I'm running the service exposed to the internet with reverse proxy and don't want to generate a large amount of traffic from unknown users streaming these vods.

**My Solution**: In an "alternative" solution I was thinking to use Nginx proxy managers basic auth to restrict the page, but I don't know if this would affect the ganymede-api or ganymede-nginx.

Thanks for your reply.

**Comment by Zibbp** [comment]
Hi,
The frontend container has an optional environment variable `REQUIRE_LOGIN` that if set to `true` will require users to be authenticated to view videos. More information can be found in the [release notes](https://github.com/Zibbp/ganymede-frontend/releases/tag/v1.1.9) that introduced this feature.

Does this help?

**Comment by szotyo-sys** [comment]
Yes, that's awesome thanks, furthermore this existing feature could get some upgrade or maybe via introducing a new optional variable to hide the complete "Recently Archived, Channels, Playlists" for unauthorized requests and just the login/register page is available.
